lib/pages/search/service/info_provider.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:acs_upb_mobile/pages/people/model/person.dart';
+import 'package:acs_upb_mobile/pages/people/service/person_provider.dart';
+import 'package:acs_upb_mobile/widgets/toast.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:provider/provider.dart';
+
+class InfoProvider{
```

Comment:
Use the provider for each type of info. Just create a method called "search(String query)" in each provider you want to use (e.g. PersonProvider).